src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.cs

```diff
@@ -143,5 +144,8 @@ public static bool IsNonZeroLowerBoundArraySupported
         // System.Security.Cryptography.Xml.XmlDsigXsltTransform.GetOutput() relies on XslCompiledTransform which relies
         // heavily on Reflection.Emit
         public static bool IsXmlDsigXsltTransformSupported => !PlatformDetection.IsUap;
+
+        // See https://github.com/dotnet/corefx/pull/37358#discussion_r280442394
```

Comment:
Nit: The property name is self-describing. I do not think this needs a link to the discussion.